README.md

```diff
@@ -63,6 +59,11 @@ class AppDelegate: NSObject, UIApplicationDelegate {
         // You can optionally provide a Region to set the Region for your Workspace:
         CustomerIO.initialize(siteId: "YOUR SITE ID", apiKey: "YOUR API KEY", region: Region.EU)
 
+        // You can optionally configure the behavior of the SDK: 
```

Comment:
I think it would be a value add to customers to have a link to the docs showing them *all* of the configuration options they have available to them.